packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -228,6 +245,74 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
   fs.writeFileSync(argv.output_file, JSON.stringify(releases, null, 2))
 }
 
+async function initializeRGImplementation(
+  releaseGoldInstance: any,
+  from: string,
+  initializerAbiRG: string,
+  transferImplOwnershipAbiRG: string
+) {
+  // We need to fund the RG implementation instance in order to initialize it.
+  console.info('Funding ReleaseGold implementation so it can be initialized...')
+  await retryTx(web3.eth.sendTransaction, [
+    {
+      from,
+      to: releaseGoldInstance.address,
+      value: 1,
+    },
+  ])
+  // Initialize and lock ownership of RG implementation
+  const implementationInitArgs = [
+    Math.round(new Date().getTime() / 1000),
+    0,
+    1,
+    1,
+    1,
+    false, // should not be revokable
+    '0x0000000000000000000000000000000000000001',
+    NULL_ADDRESS,
+    NULL_ADDRESS,
+    true, // subjectToLiquidityProivision
+    0,
+    false, // canValidate
+    false, // canVote
+    '0x0000000000000000000000000000000000000001',
+  ]
+  const implementationTransferOwnershipArgs = ['0x0000000000000000000000000000000000000001']
+  checkFunctionArgsLength(implementationInitArgs, initializerAbiRG)
+  checkFunctionArgsLength(implementationTransferOwnershipArgs, transferImplOwnershipAbiRG)
+  const implInitCallData = web3.eth.abi.encodeFunctionCall(initializerAbiRG, implementationInitArgs)
+  const transferImplOwnershipCallData = web3.eth.abi.encodeFunctionCall(
```

Comment:
maybe section this by 
1. initialize
2. transfer ownership
just a thought but I can see the opposing argument